src/Pages/Cart.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
Reloading the page cleans the cart. Even the I have not removed. The cart loses its state